### PR TITLE
Integrate with pymongo

### PR DESCRIPTION
**Pull request by tjzo**
Just integrate with pymongo

### CODE REVIEW

**Comment by reyang** [comment]
Thanks for the contribution! It looks good to me in general.

Would you move the package to `https://github.com/census-instrumentation/opencensus-python/tree/master/contrib`?

File: README.rst [hunk 1]
```diff
@@ -259,6 +259,7 @@ OpenCensus supports integration with popular web frameworks, client libraries an
 -  `requests`_
 -  `SQLAlchemy`_
 -  `threading`_
+-  `pymongo`_
```
Comment (reyang) [inline README.rst]: minor: sort alphabetically (case insensitive)

**Comment by reyang** [comment]
@tjzo would you consider adding this to https://github.com/census-instrumentation/opencensus-python/blob/master/opencensus/trace/config_integration.py#L20?

FYI, I plan to improve config_integration to support auto discovery of extensions, so we don't need to manually maintain the list in the future.

File: contrib/opencensus-ext-pymongo/tests/test_pymongo_trace.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        self.assertEqual(mock_tracer.current_span.attributes, expected_attrs)
+        mock_tracer.end_span.assert_called_with()
+
+
+class MockCommand(object):
+    def __init__(self, command_attrs):
+        self.command_attrs = command_attrs
+
+    def get(self, key, default=None):
+        return self.command_attrs[key]\
```
Comment (reyang) [inline contrib/opencensus-ext-pymongo/tests/test_pymongo_trace.py]: minor: put a space before `\`

**Comment by reyang** [comment]
We need to update https://github.com/census-instrumentation/opencensus-python/blob/master/nox.py#L21 for local test until we have a good solution for #544.

Sorry for the inconvenience!

**Comment by tjzo** [comment]
Sure. But the tests failed with module not found. I have no idea about that, could you please take a look?

**Comment by reyang** [comment]
> Sure. But the tests failed with module not found. I have no idea about that, could you please take a look?

That's the nox.py caveat! Please refer to my previous comment.

**Comment by tjzo** [comment]
> > Sure. But the tests failed with module not found. I have no idea about that, could you please take a look?
> 
> That's the nox.py caveat! Please refer to my previous comment.

Thx, I see that.

**Comment by reyang** [comment]
Thank you @tjzo!

**Pull request merged**